### ISSUE TITLE
[0.1.45] [1.16.5] Fossura doesn't steal growth ticks from wheat that is bonemealed

### ISSUE THREAD

**Issue by Not-February** [bafomdad/uniquecrops#54]
Mostly it seems to work as intended but it seems a bit strange that you can bonemeal the wheat near the Fossura and it will simply grow the wheat instead of allowing the Fossura to steal that growth to speed up the quarry.

**Comment by bafomdad** [comment]
it is odd, but the way the growth tick stealing works, is the event that lets one interrupt the growth tick is subscribed to the crop's normal growing attempts-- the bonemeal would call on a completely different method to calculate how much it's allowed to grow, as long as it's allowed to use bonemeal.

so it's a bit of a misnomer to say it steal growth ticks, and probably more accurate to state that it steals _natural_ growth ticks, as done in-world through random block ticks.

**Comment by Not-February** [comment]
Fair enough. Just a thought - maybe you could cancel the bonemeal event on wheat within range of a Fossura to avoid confusion?
